CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 0.1.9
+
+Ensure build of project before publishing to npm #127
```

Comment:
This one #124 doesn't appear in the Changelog?

Reply:
Its in the previous release

Reply:
Oh. yes I didn't see it